task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -399,6 +406,7 @@ def test_get_context_without_ti_context_from_server(self, mocked_parse, make_ti_
 
         # Verify the context keys and values
         assert context == {
+            "conn": None,
```

Comment:
Shouldn't we be able to populate a conn accessor here?

Reply:
Done in https://github.com/apache/airflow/pull/45043/commits/2937a0cd5a16844e2604bc0bb7bc8a9698c12102